changelog/fragments/1672911993-v2-directory.yaml

```diff
@@ -0,0 +1,33 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: breaking-change
```

Comment:
I would maybe categorize this as "upgrade" since it isn't a breaking change for using the agent, it is just notable.

To me a breaking change means some user action is required when upgrading to 8.6, and this isn't the case here.